arguments/SPIR-V.argdown

```diff
@@ -16,12 +16,16 @@ Relevant issues:
 [SPIR-V]: We should define a safe subset of SPIR-V and
 [accept it directly](https://github.com/gpuweb/gpuweb/issues/44) in WebGPU implementations.
 
+
+
 <SPIR-V is too low level>: SPIR-V is meant to be fed into drivers,
 while WebGPU needs to be able to generate HLSL and MSL, which are higher level.
 This makes some translation paths inefficient and awkward to implement.
   + it has a lot of instructions, some with [undefined behavior](https://github.com/gpuweb/gpuweb/issues/34).
   + it represents control flow as a graph (CFG),
     which forces us to recover the structure when generating HLSL and MSL.
+  - it lets everybody innovate on the shader language itself. It worked very well for WASM and JS/Babel.
+  - Lower level technologies are more perennial. Example: Rust, ASM, C
```

Comment:
I think we should remove this argument. Rust is not perennial yet by any means (if I understand the term correctly).
ASM and C have different stories altogether (i.e. C wasn't "low level" at all when it came out). Etc